### PR TITLE
*: replace capture ID with address in metrics

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Replace capture ID with address in metrics

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has exported variable/fields change
 - Has persistent data change

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -1,4 +1,46 @@
...
+    {
+      "name": "DS_TEST-CLUSTER",
+      "label": "test-cluster",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
+    }
+  ],
+  "__requires": [
```
Comment (overvenus) [inline metrics/grafana/ticdc.json:12]: I add these two back, are they compatible with tiup? @lichunzhu

File: metrics/grafana/ticdc.json [hunk 2]
```diff
@@ -1,4 +1,46 @@
 {
+  "__inputs": [
+    {
+      "name": "DS_TEST-CLUSTER",
```
Comment (lichunzhu) [inline metrics/grafana/ticdc.json:4]:
```suggestion
      "name": "${DS_TEST-CLUSTER}",
```

Comment (overvenus) [reply metrics/grafana/ticdc.json:4]: It is generated from Grafana.

Comment (lichunzhu) [reply metrics/grafana/ticdc.json:4]: But tiup use `${DS_TEST-CLUSTER}` to do the replacement.
https://github.com/pingcap/tiup/blob/a33f0971969b5cce3d56784c760af3269368461d/templates/scripts/run_grafana.sh.tpl#L18

Comment (lichunzhu) [reply metrics/grafana/ticdc.json:4]: never mind, I think this should work

File: metrics/grafana/ticdc.json (hunk 1 above)
Comment (lichunzhu) [reply metrics/grafana/ticdc.json:12]: I think they are compatible with tiup. @overvenus

File: scripts/check-copyright.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 copyright="// Copyright $(date '+%Y') PingCAP, Inc."
 
-result=$(find ./ -name "*.go" ! -path ".//vendor/*" | grep -v '.pb.go' | while read file_path; do
+result=$(find ./ -name "*.go" ! -path ".//vendor/*" | grep -vE '.pb.go|vendor/' | while read file_path; do
```
Comment (leoppro) [inline scripts/check-copyright.sh:3]:
```suggestion
result=$(find ./ -name "*.go" | grep -vE '.pb.go|vendor/' | while read file_path; do
```

File: cdc/processor.go [hunk 4]
```diff
@@ -265,7 +266,8 @@ func (p *processor) wait() {
 	err := p.wg.Wait()
 	if err != nil && errors.Cause(err) != context.Canceled {
 		log.Error("processor wait error",
-			zap.String("captureID", p.captureID),
+			zap.Reflect("captureid", p.captureInfo.ID),
```
Comment (amyangfei) [inline cdc/processor.go:269]:
```suggestion
			zap.String("captureid", p.captureInfo.ID),
```

File: cdc/processor.go [hunk 5]
```diff
@@ -265,7 +266,8 @@ func (p *processor) wait() {
...
+			zap.Reflect("captureaddr", p.captureInfo.AdvertiseAddr),
```
Comment (amyangfei) [inline cdc/processor.go:270]:
```suggestion
			zap.String("captureaddr", p.captureInfo.AdvertiseAddr),
```

**Pull request merged**